scripts/node_builder_glibc_env.sh

```diff
@@ -5,8 +5,15 @@
 #
 set -eu
 
-export DEBIAN_FRONTEND=noninteractive
-apt update -y && apt install -y --no-install-recommends curl g++ python3 make gcc git ca-certificates npm
-# apt has node v10 by default, so we need to add newer version to run node-gyp
-curl -fsSL https://deb.nodesource.com/setup_16.x | bash -
-apt install -y --no-install-recommends nodejs
+yum update -y && yum install -y curl python3 make git ca-certificates npm
+# yum has node v10 by default, so we need to add newer version to run node-gyp
+curl -fsSL https://rpm.nodesource.com/setup_16.x | bash -
+yum install -y nodejs
+# node-gyp needs python 3.7+, so we need to install it
```

Comment:
On my env python installation takes around 3 minutes. Entire node_builder_glibc_env.sh script takes around 4.5 minutes

Reply:
Nice. Not too much. Even with that - we can eliminate this as well with https://github.com/Granulate/gprofiler/pull/597 because that PR sets up python 3.10 on CentOS 7, we'll be able to base off that image them. We can do that later